chsdi/tests/integration/__init__.py

```diff
@@ -32,6 +35,19 @@ def tearDown(self):
         del self.testapp
         del self.grids
 
+    @contextmanager
+    def getSession(self):
+        session = scoped_session(sessionmaker())
+        yield session
+        session.close()
+
+    def getRandomFeatureId(self, bodId):
+        models = models_from_bodid(bodId)
+        with self.getSession() as session:
+            query = session.query(models[0]).limit(1)
+            reslt = query.one()
```

Comment:
it's random from a user point of view, as in I don't care just give me some id. You'd need ORDER BY RANDOM() to really get a different value every time.